ui/src/components/TaskRunsCard/index.jsx

```diff
@@ -337,10 +329,34 @@ export default class TaskRunsCard extends Component {
         <div>
           <CardContent
             classes={{
-              root: classNames(classes.cardContent, {
-                [classes.collapsedCard]: !showMore && run,
-              }),
+              root: classNames(classes.cardContent),
             }}>
+            <MobileStepper
```

Comment:
Moved stepper to the top of the card to improve navigation. Details are shown in full, so in order to move to next/previous run, navigation is displayed in the fixed position on top